modules/system-disaster-recovery/src/main/java/org/apache/ignite/internal/disaster/system/SystemDisasterRecoveryManagerImpl.java

```diff
@@ -131,9 +119,14 @@ public CompletableFuture<Void> stopAsync(ComponentContext componentContext) {
         return nullCompletedFuture();
     }
 
+    @Override
+    public void saveClusterState(ClusterState clusterState) {
+        storage.saveClusterState(clusterState);
+    }
+
     @Override
     public void markNodeInitialized() {
-        vaultManager.put(new ByteArray(NODE_INITIALIZED_VAULT_KEY), new byte[]{1});
+        storage.markNodeInitialized();
```

Comment:
What is an "initialized" node?

Reply:
This is a node that saw initial configuration

Reply:
Probably the terminology is not the best here. We need to make sure that a CMG repair only happens in a cluster where the initial configuration is already in the Metastorage (and hence it is not needed in the CMG anymore, so we can pass `null` for the initial configuration on reinit).

Reply:
Why do we call the node as "initialized" then? Saving something to Meta Storage is a cluster attribute. Also why do we need to store this state locally? Is Meta Storage not available?

Reply:
Renamed it to `markInitConfigApplied()`